registration-processor/stage-groups/stage-group-1/Dockerfile

```diff
@@ -33,23 +37,28 @@ ENV is_glowroot_env=${is_glowroot}
 # environment variable to pass artifactory url, at docker runtime
 ENV artifactory_url_env=${artifactory_url}
 
+ENV stage_group_jar_name_env=${stage_group_jar_name}
```

Comment:
do we need this env? can't we use this name directly in copy command since it will not change for this group

Reply:
With this evn, we can reuse the same docker file without any change for different stage group, just need to pass the build argument for each during docker build.

Reply:
But anyway we need to copy paste the docker file rt? Since it is not runtime and only build time, let not have the env, also the build command will be complicated rt?

Reply:
Ok.. instead of making change to the docker build command let's change in dockerfile in each stage.

Reply:
Pushed the changes.